### PR TITLE
Feature/draw gameover

### PR DESCRIPTION
**Pull request by mjwoon**
Fix vertical line for 2player mode

### CODE REVIEW

File: src/screen/GameScreen.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package screen;
 
+import java.awt.*;
```
Comment (logpacket) [inline src/screen/GameScreen.java]: This is unnecessary change. please remove it.

**Review by logpacket — CHANGES_REQUESTED** [review]

File: src/screen/GameScreen.java (hunk 1 above)
Comment (mjwoon) [reply src/screen/GameScreen.java]: Fixed unnecessary changes

**Review by logpacket — APPROVED** [review]
